packages/wallets/coin98/src/adapter.ts

```diff
@@ -21,6 +21,7 @@ interface Coin98Wallet {
     request(params: { method: string; params: string | string[] | unknown }): Promise<{
         signature: string;
         publicKey: string;
+        sig?: string;
```

Comment:
Why is there a `sig` and `signature` field? Why not just use the `signature` field?